src/main/io/serial_cli.c

```diff
@@ -2252,18 +2252,28 @@ static void cliFlashInfo(char *cmdline)
             layout->sectors, layout->sectorSize, layout->pagesPerSector, layout->pageSize, layout->totalSize, flashfsGetOffset());
 }
 
+
 static void cliFlashErase(char *cmdline)
 {
     UNUSED(cmdline);
+    uint32_t i;
 
-    cliPrintf("Erasing...\r\n");
+    cliPrintf("Erasing, please wait ... \r\n");
```

Comment:
We should probably wrap the output in here in `#ifdef CLI_MINIMAL_VERBOSITY`, in order to shave off a couple more bytes for those F1 targets that need it.